packages/fxa-payments-server/src/routes/Checkout/index.tsx

```diff
@@ -442,56 +461,41 @@ export const Checkout = ({
                     />
                   </Suspense>
                 </div>
-
-                <>
-                  <Localized id="pay-with-heading-card-or">
-                    <p className="pay-with-heading">Or pay with card</p>
-                  </Localized>
-                  <AcceptedCards />
-                </>
               </>
             )}
 
-            {!paypalScriptLoaded && (
-              <>
-                <Localized id="pay-with-heading-card-only">
-                  <p className="pay-with-heading">Pay with card</p>
-                </Localized>
-
-                <AcceptedCards />
-              </>
-            )}
-          </>
-
-          <div>
-            <Localized id="new-user-card-title">
-              <div className="label-title">Enter your card information</div>
-            </Localized>
-
-            <PaymentForm
-              {...{
-                submitNonce,
-                onSubmit: onStripeSubmit,
-                onChange,
-                submitButtonL10nId: 'new-user-submit',
-                submitButtonCopy: 'Subscribe Now',
-                shouldAllowSubmit:
-                  checkboxSet &&
-                  validEmail !== '' &&
-                  !accountExists &&
-                  !invalidEmailDomain &&
-                  emailsMatch,
-
-                inProgress,
-                validatorInitialState,
-                confirm: false,
-                submit: true,
-                plan: selectedPlan,
-                onMounted: onFormMounted,
-                onEngaged: onFormEngaged,
-                promotionCode: coupon?.promotionCode,
-              }}
-            />
+            <AcceptedCards />
+
+            <div>
+              <Localized id="new-user-card-title">
```

Comment:
Removed the h3 to keep consistent for all headers of similar type/placement. This will have to be reviewed in 3.0 with Design whether some of these titles should be headers for improved SEO and a11y.